backend/src/routes/notes.ts

```diff
@@ -0,0 +1,23 @@
+import express, { NextFunction, Request, Response } from "express";
+import { Note } from "../models/notes";
+
+const router = express.Router();
+
+router.get("/notes/:id", async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
It's good for now but once the get user PR gets approved be sure to add the authorization middleware. These routes should be user protected.